CHANGELOG.md

```diff
@@ -1,289 +1,221 @@
+-	InputField.maxLength deprecated since 4.27. Use FieldInput.maxLength instead.
+-	InputField.minLength deprecated since 4.27. Use FieldInput.minLength instead.
+-	InputField.name deprecated since 4.27. Use FieldInput.name instead.
+-	InputField.required deprecated since 4.27. Use FieldInput.required instead.
+-	InputField.submittable deprecated since 4.27. Use FieldInput.submittable instead.
+-	InputField.type deprecated since 4.27. Use FieldInput.dataType instead.
+-	InputField.updating deprecated since 4.27. Use FieldInput.updating instead.
+-	InputField.valid deprecated since 4.27. Use FieldInput.valid instead.
+-	InputField.value deprecated since 4.27. Use FieldInput.value instead.
+-	InputField.visible deprecated since 4.27. Use FieldInput.visible instead.
+-	InputFieldGroup deprecated since 4.27. Use GroupInput instead.
+-	InputFieldGroup.description deprecated since 4.27. Use GroupInput.description instead.
+-	InputFieldGroup.evaluatedVisibilityExpression deprecated since 4.27. Use GroupInput.visible instead.
+-	InputFieldGroup.inputFields deprecated since 4.27. Use GroupInput.inputs instead.
+-	InputFieldGroup.label deprecated since 4.27. Use GroupInput.label
+-	InputFieldGroup.state deprecated since 4.27. Use GroupInput.state
+-	InputFieldGroup.visibilityExpression deprecated since 4.27. Use Element.visibilityExpression
+-	InputFieldGroup.visible deprecated since 4.27. Use GroupInput.visible
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	FeatureTable.clearHighlights deprecated since version 4.25. Use highlightIds.removeAll() instead.
+-	FeatureTable.clearSelection deprecated since version 4.25. Use highlightIds.removeAll() instead.
+-	FeatureTable.deselectRows deprecated since 4.25. Use highlightIds.remove() instead.
+-	FeatureTable.selection-change deprecated since version 4.25. Listen for changes on highlightIds instead.
+-	FeatureTable.highlightOnRowSelectEnabled deprecated since version 4.25. Use highlightEnabled instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	FeatureTable.selectRows deprecated since 4.25. Use highlightIds.add() instead.
+-	FeatureTableViewModel.clearHighlights deprecated since version 4.25. Use highlightIds.removeAll() instead.
+-	FeatureTableViewModel.clearSelection deprecated since version 4.25. Use highlightIds.removeAll() instead.
+-	FeatureTableViewModel.highlightOnRowSelectEnabled deprecated since version 4.25. Use highlightEnabled instead.
+-	FeatureTableViewModel.selectRows deprecated since 4.25. Use highlightIds.add() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
```

Comment:
Can you copy the list of deprecations from the production release notes? Seeing a lot of duplicates here that are not in the production release notes list

Reply:
Wow, I see that now. I'll fix it. I created the PR several weeks ago and clearly didn't look close enough. I also see comments in there like this:

-	Popup.maxInlineActions deprecated TODO: DOCUMENT

Reply:
Production deprecations have been re-added. Sorry about that.